README.txt

```diff
@@ -0,0 +1,52 @@
+ibmDFDLCrossTester
+
+This is a test rig that drives Daffodil TDML tests against IBM DFDL enabling
+cross testing of tests easily against both Daffodl and the IBM DFDL implementation.
+
+Requirements
+
+IBM DFDL - a developer edition is available. 
+Daffodil - snapshot of 2.3.0 development branch as of at least 2018-10-30 or a newer release
+
+How to Install
+
+Edit the build.sbt file so that the daffodil dependency requests the right version of Daffodil. 
+If using a development snapshot, the dependency will end with something like "2.3.0-SNAPSHOT", and 
+you must insure that you have a locally published snapshot of that branch.
+
+Download and install the developer edition of IBM. (Tested using ace-11.0.0.1)
+In it is a DFDL jar that you can open up.
+
+Copy the IBM DFDL jars into the 'lib' subdirectory of this project.
+
+You must also copy the samples/dfdlsample_java.jar into the 'lib' subdirectory, as that also has class files in it that this test rig uses.  
+
+The resulting directory looks like this.
+
+lib
+├── dfdlsample_java.jar
+├── emf.common_2.6.0.jar
+├── emf.ecore_2.6.1.jar
+├── emf.ecore.xmi_2.5.0.jar
+├── gpb.jar
+├── ibm-dfdl.jar
+├── icu4j-charsets.jar
+├── icu4j.jar
+├── scd.jar
+└── xsd_2.6.0.jar
+
```

Comment:
Insert in here instructions on copying the sample data files into src/test/resources (company.xml, .xsd, and .txt).

I think the import/included IBMDefined/RecordSeparatedFieldFormat.xsd will be picked up from the sample jar, so needn't be copied.

Reply:
Done